docs/source/release_notes/release-notes.rst

```diff
@@ -44,12 +50,14 @@ develop
     *    - Type
          - Change
          
-    *    - |improved|
+    *    - |metrics|
```

Comment:
`|metrics|` and `|logs|` aren't quite at the same level of abstraction as `|new|`, `|fixed|` etc. Consider that if we used the internal structured changelog framework, then metrics and logs would be tags. It's OK that we use changetypes for this, for lack of a better alternative, but the type of the change should be maintained. So I'd write this change `|improved| |metrics|`.